buildSrc/src/main/java/Dependencies.kt

```diff
@@ -4,7 +4,7 @@
 
 // Synchronized version numbers for dependencies used by (some) modules
 object Versions {
-    const val kotlin = "1.3.31"
+    const val kotlin = "1.3.40"
     const val coroutines = "1.2.1"
```

Comment:
Please update [coroutines to 1.2.2](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.2.2) too.

Reply:
👍